ui/app/plugins/page.tsx

```
@@ -1,392 +1,394 @@
-							})}
-						</div>
-					</section>
+                      <div className="space-y-2">
+                        <Button asChild variant="outline" className="w-full">
+                          <Link href={plugin.url} target="_blank">
+                            <Code className="mr-1 h-4 w-4" />
+                            Source Code
+                          </Link>
+                        </Button>
+                        <Button asChild variant="outline" className="w-full">
+                          <Link href={plugin.url + '/README.md'} target="_blank">
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)